src/tasks/leveling.ts

```diff
@@ -392,46 +450,108 @@ export const LevelingQuest: Quest = {
+        have(abstractionXpItem) ||
+        have(abstractionXpEffect) ||
+        storageAmount(abstractionXpItem) === 0 ||
+        get("instant_saveAbstraction", false),
+      do: (): void => {
+        takeStorage(abstractionXpItem, 1);
+        chew(abstractionXpItem, 1);
+      },
+      limit: { tries: 1 },
+    },
+    {
+      name: "Pull Repaid Diaper",
       completed: () =>
         get("_roninStoragePulls").split(",").length >= 5 ||
         get("_roninStoragePulls")
           .split(",")
-          .includes(toInt($item`abstraction: category`).toString()) ||
-        have($item`abstraction: category`) ||
-        have($effect`Category`) ||
-        storageAmount($item`abstraction: category`) === 0 ||
-        get("instant_saveAbstraction", false),
+          .includes(toInt($item`repaid diaper`).toString()) ||
+        have($item`repaid diaper`) ||
+        storageAmount($item`repaid diaper`) === 0 ||
+        !get("instant_experimentPulls", true),
+      do: (): void => {
+        takeStorage($item`repaid diaper`, 1);
+      },
+      limit: { tries: 1 },
+    },
+    {
+      name: "Pull Beastly Trousers",
+      completed: () =>
+        get("_roninStoragePulls").split(",").length >= 5 ||
+        get("_roninStoragePulls")
+          .split(",")
+          .includes(toInt($item`Great Wolf's beastly trousers`).toString()) ||
+        get("_roninStoragePulls")
+          .split(",")
+          .includes(toInt($item`repaid diaper`).toString()) ||
+        have($item`Great Wolf's beastly trousers`) ||
+        storageAmount($item`Great Wolf's beastly trousers`) === 0 ||
+        !get("instant_experimentPulls", true),
+      do: (): void => {
+        takeStorage($item`Great Wolf's beastly trousers`, 1);
+      },
+      limit: { tries: 1 },
+    },
+    {
+      name: "Pull Stick Knife",
+      completed: () =>
+        get("_roninStoragePulls").split(",").length >= 5 ||
+        get("_roninStoragePulls")
+          .split(",")
+          .includes(toInt($item`Stick-Knife of Loathing`).toString()) ||
+        have($item`Stick-Knife of Loathing`) ||
+        storageAmount($item`Stick-Knife of Loathing`) === 0 ||
+        !myPrimestat() === $stat`Muscle` ||
+        !get("instant_experimentPulls", true),
+      do: (): void => {
+        takeStorage($item`Stick-Knife of Loathing`, 1);
+      },
+      limit: { tries: 1 },
+    },
+    {
+      name: "Pull Marble Soda",
+      completed: () =>
+        get("_roninStoragePulls").split(",").length >= 5 ||
+        get("_roninStoragePulls")
+          .split(",")
+          .includes(toInt($item`tobiko marble soda`).toString()) ||
+        have($item`tobiko marble soda`) ||
+        storageAmount($item`tobiko marble soda`) === 0 ||
+        !get("instant_experimentPulls", true),
       do: (): void => {
-        takeStorage($item`abstraction: category`, 1);
-        chew($item`abstraction: category`, 1);
+        takeStorage($item`tobiko marble soda`, 1);
       },
```

Comment:
I don't think these experimental pulls should be added to main in this manner or at this stage